src/ontology/cob.Makefile

```diff
@@ -36,88 +37,52 @@ products/:
-$(ONT)-base.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC)
-	$(ROBOT_RELEASE_IMPORT_MODE_BASE) \
-	$(SHARED_ROBOT_COMMANDS) \
-	annotate --link-annotation http://purl.org/dc/elements/1.1/type http://purl.obolibrary.org/obo/IAO_8000001 \
-		--ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		--output $@.tmp.owl && mv $@.tmp.owl $@
-
-cob-base-reasoned.owl: cob-base.owl
-	$(ROBOT) remove --input $< --select imports --trim false \
-		reason -r HERMIT \
-		annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		--output $@.tmp.owl && mv $@.tmp.owl $@
-
-cob-examples-reasoned.owl: cob-base.owl $(COB_EXAMPLES)
-	$(ROBOT) remove --input $< --select imports --trim false \
-		merge $(patsubst %, -i %, $^) \
-		reason -r HERMIT \
-		annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		--output $@.tmp.owl && mv $@.tmp.owl $@
-
-# TSV export (may depend on dev version of robot export)
-cob.tsv: cob.owl
-	$(ROBOT) export -i $<  -c "ID|ID [LABEL]|definition|subClassOf [ID NAMED]|subClassOf [LABEL NAMED]|subClassOf [ID ANON]|subClassOf [LABEL ANON]" -e $@
-#	$(ROBOT) export -i $< --entity-select NAMED -c "ID|ID [LABEL]|definition|subClassOf [ID]|subClassOf [LABEL]|subClassOf [ID ANON]|subClassOf [LABEL ANON]" -e $@
-
-# -- BRIDGING AXIOMS TO OBO ROOTS --
-#
-#  the source file is cob-to-external.tsv
-#
-#  OWL is generated from this
-#
 
-$(TMPDIR)/cob-to-external.sssom.owl: $(COMPONENTSDIR)/cob-to-external.tsv | $(TMPDIR)
-	sssom convert $< --output-format owl -o $@
+########################################
+# -- MAIN RELEASE PRODUCTS --
+########################################
 
-$(COB_TO_EXTERNAL): $(TMPDIR)/cob-to-external.sssom.owl
-	$(ROBOT) merge -i $< \
+$(ONT)-edit.owl: $(TMPDIR)/cob-root.tsv $(COMPONENTSDIR)/obsolete.tsv
+	$(ROBOT) template \
+	$(foreach X,$(filter %.tsv,$^),--template $(X)) \
+	reason -r HERMIT \
 	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-	convert -f owl -o $@
+	$(ANNOTATE_ONTOLOGY_METADATA) \
+	--output $@.tmp.owl && mv $@.tmp.owl $@
 
-$(TMPDIR)/cob-annotations.ttl: $(COB_TO_EXTERNAL) $(SPARQLDIR)/external-links.rq | $(TMPDIR)
-	$(ROBOT) query --input $< --query $(word 2,$^) $@
+# COB "Full"
+$(ONT).owl: $(TMPDIR)/cob-full.tsv $(COMPONENTSDIR)/obsolete.tsv
+	$(ROBOT) template \
+	$(foreach X,$(filter %.tsv,$^),--template $(X)) \
+	reason -r HERMIT \
+	annotate --ontology-iri $(ONTBASE).owl $(ANNOTATE_ONTOLOGY_VERSION) \
+	$(ANNOTATE_ONTOLOGY_METADATA) \
+	--output $@.tmp.owl && mv $@.tmp.owl $@
 
-$(COB_ANNOTATIONS): $(TMPDIR)/cob-annotations.ttl
-	$(ROBOT) annotate --input $< \
-	--ontology-iri "http://purl.obolibrary.org/obo/cob/$@" \
-	--annotation owl:versionInfo $(TODAY) \
-	--output $@
+$(ONT)-base.owl: $(ONT).owl $(TMPDIR)/cob-base.tsv $(COMPONENTSDIR)/obsolete.tsv
+	$(ROBOT) template --input $< \
+	$(foreach X,$(filter %.tsv,$^),--template $(X)) \
+	annotate --link-annotation http://purl.org/dc/elements/1.1/type http://purl.obolibrary.org/obo/IAO_8000001 \
```

Comment:
shouldnt we use dc terms (I know this was always not the case)?

Reply:
Sorry, I always get confused by DC terms/elements.

Reply:
Looks like this is copied from ODK, which is using DC elements: https://github.com/INCATools/ontology-development-kit/blob/b9285edc0e161e87d33cdbfeacb635e5883fca37/template/src/ontology/Makefile.jinja2#L1144

So I'm going to leave it this way for now.